docs/tutorials/genome.ipynb

```diff
@@ -0,0 +1,267 @@
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "MUXex9ctTuDB"
+      },
+      "source": [
+        "## Setup"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "IqR2PQG4ZaZ0",
+        "colab": {}
+      },
+      "source": [
+        "try:\n",
+        "  %tensorflow_version 2.x\n",
+        "except Exception:\n",
+        "  pass\n",
+        "!pip install tensorflow-io"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "bkF2WtCMaJ-3",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "import tensorflow_io as tfio\n",
+        "import tensorflow as tf"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "6wkjlql3cOy0",
+        "colab_type": "text"
+      },
+      "source": [
+        "## FASTQ Data\n",
+        "FASTQ is a common genomics file format that stores both sequence information in addition to base quality information.\n",
+        "\n",
+        "First, let's download a sample `fastq` file."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "yASvppCxceBu",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "# Download some sample data:\n",
+        "!curl -OL https://raw.githubusercontent.com/tensorflow/io/master/tests/test_genome/test.fastq"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3zekWXlVdprb",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Read FASTQ Data\n",
+        "Now, let's use `tfio.genome.read_fastq` to read this file (note a `tf.data` API coming soon)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "vl761cHTc7N1",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "fastq_data = tfio.genome.read_fastq(filename=\"test.fastq\")\n",
+        "print(fastq_data.sequences)\n",
+        "print(fastq_data.raw_quality)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "qxHjVKXzdx5W",
+        "colab_type": "text"
+      },
+      "source": [
+        "As you see, the returned `fastq_data` has `fastq_data.sequences` which is a string tensor of all sequences in the fastq file (which can each be a different size) along with `fastq_data.raw_quality` which includes Phred encoded quality information about the quality of each base read in the sequence.\n",
+        "\n",
+        "### Quality\n",
+        "We can use a helper op to convert this quality information into probabilities if we are interested."
```

Comment:
Google dev docs style prefers second person "you" instead of "we": https://developers.google.com/style/person

Reply:
Makes sense--done!